tests/pass/concurrency/linux-futex.rs

```diff
@@ -206,6 +208,47 @@ fn wait_wake_bitset() {
     t.join().unwrap();
 }
 
+const FREE: i32 = 0;
+const HELD: i32 = 1;
+fn concurrent_wait_wake() {
+    static FUTEX: AtomicI32 = AtomicI32::new(0);
+    for _ in 0..100 {
```

Comment:
100 times is a lot.

What about reducing this (to make the test not take quite as long), and bumping up the preemption rate for this test?

Reply:
FWIW, the test currently already [fails in the 3rd iteration](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=31134a8369681cade276e063328218ab),

Reply:
Is this even about the preemption rate? If it is about picking 1 out of 2 entries in the store buffer, then 20 rounds make the probability of accidentally passing the test 2^-20, which seems easily good enough.

Reply:
> We do need to get the wake to run before the wait

So preemption is actually harmful? Should we set the preemption rate to 0?

Reply:
I think leaving it as-is is fine. Having preemption also demonstrates that the critical sections inside futex syscalls are happening atomically